src/content/protocols/compound-v3.md

```diff
@@ -0,0 +1,225 @@
+
+Compound-v3 is a lending protocol that accepts a base asset as liquidity and allows borrowing this bas asset with a variety of other assets as collateral. Multiple base assets are supported such as USDC, WETH, USDT, wstETH, and USDS. Each base asset represents an isolated lending market managed by a separate instance of the protocol. Compound governance is able to update various parameters for each of these markets.
+
+# Overview
+
+## Chain
+
+Compound-v3 is deployed on various chains. This review is based on the Ethereum mainnet deployment.
+
+> Chain score: Low
+
+## Upgradeability
+
+The Compound-v3 protocol is fully upgradeable allowing for the update of governance and markets logic and state (specifically the Governance and Comet implementation contracts). This can result in the loss of funds or unclaimed yield as well as lead to other changes in the expected performance of the protocol.
+
+The permission to upgrade the protocol is controlled by an onchain governance system with COMP token holders submitting and voting on respective proposals. A multisig account, the ProposalGuardian, has the permission to cancel proposals to mitigate the risk of malicious or otherwise unintended proposals. This role can potentially be abused to censor proposals.
+
+Furthermore, another multisig account, the PauseGuardian, has the permission to pause markets, disabling depositing and withdrawing assets, if suspicious activity is detected. This role can potentially be abused to freeze funds and unclaimed yield in the protocol.
+
+> Upgradeability score: High
+
+## Autonomy
+
+The compound-v3 protocol relies on a Chainlink oracle feed to price collateral and base assets in the system. The protocol does not validate asset prices returned by Chainlink or offer a fallback oracle mechanism. The replacement of a stale or untrusted oracle feed requires a Compound governance vote with a delay (see Exit Window).
+
+The Chainlink oracle system itself is upgradeable without decentralized ownership over those permissions. This dependency thus introduces centralization risk in the Compound-v3 protocol.
+
+> Autonomy score: High
+
+## Exit Window
+
+Permissions, including protocol upgrades, are controlled by an onchain governance system. COMP holders are able to create new proposals (requires 25,000 COMP) and vote on proposals (at least 400,000 votes are required for a valid proposal). A minimum voting period of 3 days is enforced as well as a delay of 2 days for the implementation of successful proposals.
+
+While this does not meet the 7-day exit window requirement, malicious or unintended proposals can be intercepted by the `ProposalGuardian` multisig account.
+
+However, both the `ProposalGuardian` and the `PauseGuardian` multisig accounts do not meet the Security Council requirements ([see below](#security-council)).
+
+> Exit Window score: High
+
+## Accessibility
+
+The frontend of Compound V3 is open source. Instructions to deploy it locally or deploy it
```

Comment:
Self-hosting option is well explained and documented in their repo

however, the FE depends on a separate backend that is accessed through https://v3-api.compound.finance/). Thus self-hosting the FE doesnt help if the BE is down

but, compound is supported on third-party apps like DeFiSaver. These apps build an acceptable backup solution in case the official FE/BE are down. pls update wording